docs/installation/upgrade/include/2.4-removed-settings.inc

```diff
@@ -0,0 +1,44 @@
+#### Settings
+
+| Setting | Notes |
+| ------- | ----- |
+| `auth_debug` | Use [[setting,log_debug]] filter instead. Example: `log_debug=category=auth`. |
+| `auth_default_realm` | Replaced by [[setting,auth_default_domain]]. |
+| `auth_stats` | |
+| `auth_worker_max_count` | Use [[link,service_configuration,service-specific process limit]]. |
+| `dict_db_config` | Berkeley DB is not supported anymore. |
+| `disable_plaintext_auth` | Replaced by [[setting,auth_allow_cleartext]]. |
```

Comment:
Maybe we should have a separate "renamed settings", since this isn't really a removal. Or looking further maybe call it "converted settings" since some settings aren't simple renames.